src/utils/stdutils.ts

```diff
@@ -0,0 +1,33 @@
+import { type StandardLibraryType } from 'services/CommonService'
```

Comment:
Please add the header in this new file and make sure that the year is set to 2024.